File: src/main/java/no/nav/melosys/eessi/models/sed/nav/Utsendingsland.java

package no.nav.melosys.eessi.models.sed.nav;

import java.util.List;
import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonInclude.Include;
import lombok.Data;


@JsonInclude(Include.NON_NULL)
@Data
public class Utsendingsland {

    private List<Arbeidsgiver> arbeidsgiver;
}

File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/models/sed/nav/Utsendingsland.kt
package no.nav.melosys.eessi.models.sed.nav

import com.fasterxml.jackson.annotation.JsonInclude

@JsonInclude(JsonInclude.Include.NON_NULL)
data class Utsendingsland(
    var arbeidsgiver: List<Arbeidsgiver>? = null
)